tests/base.py

```diff
@@ -273,6 +271,10 @@ def _setup_db(self, config):
                 'linux_path':'nowhere'}
         self.local_storage = _find_or_create_entity(self.sg, 'LocalStorage', data, keys)
 
+    def tearDown(self):
+        self.sg.delete("HumanUser", self.human_user["id"])
```

Comment:
I suggest adding a comment here explaining why we are deleting the HumanUser but keeping all the other entities that we are creating as part of the setup. Also please add a docstring to all new methods that are introduced into the codebase.